packages/react/src/app-layout/AppLayoutSidebarNav.tsx

```diff
@@ -162,7 +162,7 @@ function AppLayoutSidebarNavListItem({
 					{endElement}
 				</Link>
 
-				{isOpen && (
+				{(isOpen || isCurrentPage) && (
```

Comment:
Can we get a story showing a parent active?

Reply:
That's the current `Basic` story but I might change it to be inline with `SideNav` where basic was the first item and it had no sub-levels.

![image](https://github.com/agriculturegovau/agds-next/assets/43688949/e0de4194-1e04-4426-8332-abc3c96fe90b)

- [ ] rearrange stories

Reply:
Ah ok. I must have been looking at the old version that didn't expand, but yeah, having both is good. I would also add a story for alwaysOpen